test/modules/packages/blas/blas_helpers.chpl

```diff
@@ -54,8 +57,43 @@ proc makeHerm(A : [?Adom]) {
   }
 }
 
+
+// Make band-diagonal matrix
+proc makeBand(A:[?Adom] ?t, kl, ku)
+  where Adom.rank == 2 {
+  for i in Adom.dim(1) {
+    for j in Adom.dim(2) {
+      if (j < i - kl) || (j > i + ku) {
+        A[i,j] = (0):t;
+      }
+    }
+  }
+}
+
+// Make band-diagonal triangular matrix
+proc makeBandTriangular(A:[?Adom], k,  uplo:Uplo=Uplo.Upper)
+  where Adom.rank == 2 {
+
```

Comment:
Intended as a TODO skeleton function (in conjunction with the band/packed matrix support)